source/change-streams.rst

```diff
@@ -320,8 +364,60 @@ Presently change streams support only a subset of available aggregation stages:
 
 A driver MUST NOT throw an exception if any unsupported stage is provided, but instead depend on the server to return an error.
 
-The aggregate helper method MUST have no new logic related to the ``$changeStream`` stage. Drivers MUST be capable of handling `TAILABLE_AWAIT <https://github.com/mongodb/specifications/blob/master/source/crud/crud.rst#read>`_  cursors from the aggregate command in the same way they handle such cursors from find.
+The aggregate helper methods MUST have no new logic related to the ``$changeStream`` stage. Drivers MUST be capable of handling `TAILABLE_AWAIT <https://github.com/mongodb/specifications/blob/master/source/crud/crud.rst#read>`_  cursors from the aggregate command in the same way they handle such cursors from find.
+
+``Collection.watch`` helper
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:since: 3.6
+
+Returns a ``ChangeStream`` on a specific collection
+
+Command syntax:
+
+.. code:: typescript
+
+    {
+      aggregate: 'collectionName'
+      pipeline: [{$changeStream: {...}}, ...],
+      ...
+    }
+
+``Database.watch`` helper
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:since: 4.0
+
+Returns a ``ChangeStream`` on all collections in a database.
 
+Command syntax:
+
+.. code:: typescript
+
+    {
+      aggregate: 1
+      pipeline: [{$changeStream: {...}}, ...],
+      ...
+    }
+
+If the helper is used on the ``admin``, ``local``, or ``config`` databases, a driver MUST NOT throw a custom exception, as the server will return an error.
+
+``MongoClient.watch`` helper
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:since: 4.0
+
+Returns a ``ChangeStream`` on all collections in all databases in a cluster
+
+Command syntax:
+
+.. code:: typescript
+
+    {
```

Comment:
What database should be specified for this command?  I assume "admin" but let's be explicit.

And if it has to be "admin", is "allChangesForCluster" redundant"?  Could the server just assume "admin" database implies all databases, or is that too weird?

Reply:
Earlier, we note that `allChangesForCluster` omits the "config", "local", and "admin" databases. Given that the `Database::watch()` is intended to propagate server errors if the user attempts to watch one of those databases (https://github.com/mongodb/specifications/pull/311/files#r183389479), I'd think we wouldn't want to infer anything here. Otherwise, that might effectively make `Database::watch()` on "admin" an alternative form of `Client::watch()`.

Reply:
Yes, the database must be admin. The server design spec describes the command as:

```
db.adminCommand({
  aggregate: 1,
  pipeline: [{$changeStream: {allChangesForCluster: true, ...}}, ...],
  …
})
```

The `allChangesForCluster distinguishes it from the command that listens to all changes on a single DB